lib/api/controllers/funnelController.js

```diff
@@ -45,7 +47,13 @@ module.exports = function FunnelController (kuzzle) {
           return false;
         }
 
-        return this[requestObject.controller][requestObject.action](requestObject, connection);
+        // check if the current user is allowed to process
+        if (context.user.profile.isActionAllowed(requestObject, context) === false) {
+          deferred.reject(new ForbiddenError('Unauthorized action [' + requestObject.collection + '/' + requestObject.controller + '/' + requestObject.action + '] for user ' + context.user._id));
+          return false;
```

Comment:
should return a new rejected promise instead of ``false`` to avoid breaking the call chain and to trigger a statistics.failedRequest() event